pkg/controller/protobuf/client/client.go

```diff
@@ -232,7 +235,7 @@ func (ctrlAdapter *controllerAdapter) run(ctx context.Context) {
 
 		interval := ctrlAdapter.backoff.NextBackOff()
 
-		logger.Printf("restarting controller in %s", interval)
+		logger.Debug(fmt.Sprintf("restarting controller in %s", interval))
```

Comment:
```suggestion
		logger.Sugar().Debugf("restarting controller in %s", interval)
```

The difference is that string formatting will not be done if debug level is disabled, saving a bit of CPU time

Reply:
yeah, that's an option, it just felt like calling `Sugar` for each log line is not how it's intended to be used.

Reply:
It is, actually :) That's documented in README and their FAQ.

Another option could be to pass `*zap.SugaredLogger` instead, and call `Desugar()` where we need speed.

Reply:
:cool: